superset-frontend/plugins/legacy-preset-chart-deckgl/package.json

```diff
@@ -24,10 +24,16 @@
     "lib"
   ],
   "dependencies": {
-    "@deck.gl/aggregation-layers": "^9.0.38",
-    "@deck.gl/core": "^9.0.37",
-    "@deck.gl/layers": "^9.0.38",
-    "@deck.gl/react": "^9.1.4",
+    "@deck.gl/aggregation-layers": "^9.1.13",
```

Comment:
@plavacquery trying your suggested fix (upgraded to 9.1.13) while keeping the sub-packages, and adding luma.gl deps explicitely